File: fizzbuzz/fizzbuzz.c
#include <stdio.h>

int main(){
	int i;
	for(i = 1; i < 101; i++){
		if(i%15 == 0){
			printf("Fizzbuzz\n");
		}else if(i%3 == 0){
			printf("Fizz\n");
		}else if(i%5 == 0){
			printf("Buzz\n");
		}else
			printf("%d\n",i);
	}
}

File: fizzbuzz/fizzbuzz.py
x = 1
for x in range(1,101):
    if(x%15 == 0):
        print( "Fizzbuzz" )
    elif(x%3 == 0):
        print( "Fizz" )
    elif(x%5 == 0):
        print( "Buzz" )
    else:
        print( "%d"%(x))